2022q3/core.adoc

```diff
@@ -0,0 +1,37 @@
+=== FreeBSD Core Team
+
+Contact: FreeBSD Core Team <core@FreeBSD.org>
+
+The FreeBSD Core Team is the governing body of FreeBSD.
+
+==== Completed Items
+
+===== New Core Team Secretary
+
+All members of the Core Team express publicly their gratitude to Muhammad Moinur Rahman (bofh) for serving as the Core Team Secretary for the past two years.
+
+The Core Team approved Sergio Carlavilla (carlavilla) as the new Core Team secretary.
+
+===== Procedure to handle GDPR deletion request
+
+The Core Team has reviewed the procedure to handle GDPR deletions requests with help from Foundation lawysers.
+The document is currently being written and will be published after completion.
+
+===== New Privacy Policy
+
+The Core Team is working closely with the FreeBSD Foundation to update the Privacy Policy to properly align with current laws and practices found on similar websites such as ours.
+
+===== Bruce memorial plaque
+
+The Core Team unanimously votes to allow the memorial plaque for Bruce mentioning him as co-founder of FreeBSD.
+
+===== EuroBSDCon core team office hour
```

Comment:
Wrong indent level as this wasn't completed in Q3 but will take place in Q4.

Reply:
Are you sure about this? I read September 16 and Q3 is July-August-September, isn't it?

However, I would change "will be presenting" with "presented", since September 16 is past.

Reply:
> Are you sure about this? I read September 16 and Q3 is July-August-September, isn't it?

... I *was* sure about this when I wrote this. Brainfart, most likely.